go-controller/hybrid-overlay/cmd/hybrid-overlay-node/hybrid-overlay-node.go

```diff
@@ -108,12 +113,17 @@ func runHybridOverlay(ctx *cli.Context) error {
 	defer close(stopChan)
 	f := informers.NewSharedInformerFactory(clientset, informer.DefaultResyncInterval)
 
+	if libovsdbOvnNBClient, err = libovsdb.NewNBClient(stopChan); err != nil {
+		return fmt.Errorf("error when trying to initialize libovsdb NB client: %v", err)
+	}
+
 	n, err := controller.NewNode(
 		&kube.Kube{KClient: clientset},
 		nodeName,
 		f.Core().V1().Nodes().Informer(),
 		f.Core().V1().Pods().Informer(),
 		informer.NewDefaultEventHandler,
+		libovsdbOvnNBClient,
```

Comment:
same concern about running the ovsdb connectionf rom every node. I wonder if we could filter tables on the node  nbclient to reduce the load on ovsdb-server? @dceara?

Reply:
@trozet @astoycos while we technically *can* run the HO node code on Linux, we don't because it's run as part of ovnkube-node itself. It does run standalone on Windows nodes though, but they shouldn't be doing anything OVN database related.

Reply:
Sounds good, I will leave this then